src/control/mobile_manipulation/robots_world/hsr/hsr_analytical.launch

```diff
@@ -0,0 +1,61 @@
+<launch>
+  <!-- By default we do not overwrite the URDF. Change the following to true to change the default behavior -->
+  <arg name="load_robot_description" default="true"/>
+
+  <!-- The name of the parameter under which the URDF is loaded -->
+  <arg name="robot_description" default="robot_description"/>
+  <arg name="upload_robot_model_launch" default="$(find mobile_manipulation_rl)/robots_world/hsr/upload_hsrb.launch"/>
+
+  <!-- Load universal robot description format (URDF) -->
+  <include if="$(arg load_robot_description)" file="$(arg upload_robot_model_launch)" >
+    <remap to="$(arg robot_description)" from="/robot_description" />
+  </include>
+
+  <!-- The semantic description that corresponds to the URDF -->
+  <param name="$(arg robot_description)_semantic" textfile="$(find hsrb_moveit_config)/config/hsrb.srdf" />
+
+  <!-- Load updated joint limits (override information from URDF) -->
+  <group ns="$(arg robot_description)_planning">
+    <rosparam command="load" file="$(find hsrb_moveit_config)/config/joint_limits.yaml"/>
+  </group>
+
+  <!-- Load default settings for kinematics; these settings are overridden by settings in a node's namespace -->
+  <arg name="robot_name" default="hsrb"/>
+  <group ns="$(arg robot_description)_kinematics">
+    <rosparam command="load" file="$(find hsrb_moveit_config)/config/kinematics.yaml"/>
+    <rosparam param="robot_name" subst_value="true">"hsrb"</rosparam>
+  </group>
+
+
+
+  <!-- Start the actual move_group node/action server -->
+  <node name="move_group" launch-prefix="" pkg="moveit_ros_move_group" type="move_group" respawn="false" output="screen" args="">
+    <!-- Set the display variable, in case OpenGL code is used internally -->
```

Comment:
Please remove the template comments (keep only those that are not the standard ones).

Reply:
Removed those that do not add information